design/authenticators/authn_jwt/token_schema.md

```diff
@@ -0,0 +1,180 @@
+# Solution Design - Authn JWT Token Schema
+
+[//]: # "1. Design should be graphical-based and table-based - avoid long text explanations"
+[//]: # "2. Design documents should not be updated after implementation"
+[//]: # "3. Design decisions should be made before writing this document, and as such this document should not include options / choices"
+
+
+## Table of Contents
+[//]: # "You can use this tool to generate a TOC - https://ecotrust-canada.github.io/markdown-toc/"
+
+- [Table of Contents](#table-of-contents)
+- [Glossary](#glossary)
+- [Useful Links](#useful-links)
+- [Requirments](#requirments)
+- [Solution](#solution)
+  * [User Interface](#user-interface)
+    + [Authenticator Policy](#authenticator-policy)
+    + [Variable Values Example](#variable-values-example)
+- [Design](#design)
+  * [Class Diagrams](#class-diagrams)
+  * [Flow Explanation](#flow-explanation)
+- [Backwards Compatibility](#backwards-compatibility)
+- [Affected Components](#affected-components)
+- [Delivery Plan](#delivery-plan)
+- [Delivery Plan](#delivery-plan-1)
```

Comment:
two delivery plans? I would differentiate them somehow. Maybe according to different proposed flows?

Reply:
Good catch its mistake